news/avoid_setup_package_uninstall

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Build templates no longer install & then uninstall `conda-forge-ci-setup` package if configured for local_ci_setup
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* Removed pip from `conda install` list in build templates as already a transient dependency from conda-build
```

Comment:
We should never rely on transient deps. If one day `conda-build` does not depend on pip, then we will get an error. Any thing we explicitly use should be listed.